### PR TITLE
Bb 23 sign up page

### PR DESCRIPTION
**Pull request by brunobmv**
## Describe your changes:

created signup page. used bootstrap react forms to get dropdowns. styled the page.

## Proof that your changes work (include screenshots if frontend):
Screen Shot 2023-03-22 at 16 58 46
Screen Shot 2023-03-22 at 16 58 35
Screen Shot 2023-03-22 at 17 13 33

## Jira Issue ticket number and link:

BB-23 , https://bearbuddies.atlassian.net/browse/BB-23?atlOrigin=eyJpIjoiZDU0NzVkYzQ0MDk5NDliYWE5NWVlNjZkY2Y2MTJlMjEiLCJwIjoiaiJ9 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have DM'd the group chat that I have made a PR and need reviewers.
- [x] I haved moved the Jira issue to "Under Review"
- [x] I have run `npx eslint filename.js` on all JavaScript files I've changed in this PR and fixed all displayed errors

### CODE REVIEW

File: src/pages/Signup/signup.js
Comment (thomaspwang) [inline src/pages/Signup/signup.js:1]: image

- add slightly more space between the text
- bold the title

reference figma:
image

Comment (thomaspwang) [inline src/pages/Signup/signup.js:1]: image

the dropdown should include a dropdown button

reference figma:
image

File: src/pages/_app.js
Comment (thomaspwang) [inline src/pages/_app.js:1]: unnecessary changes

do the signup.module.css thing like we discussed

**Review by thomaspwang — CHANGES_REQUESTED** [review]
Looks great! Almost done, basically. Just like two super small changes.

**Comment by brunobmv** [comment]
Screen Shot 2023-03-22 at 23 35 07


so i did a bit of syntax gymnastics with the jsx to get the dropdown button.
for some reason, i am not being able to turn the text inside the box grey and make it smaller.
any change i make to it doesn't apply for some reason.
you did not make a comment about this so just wanted to let you know about what i thought about.

File: src/pages/Signup/Signup.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                    </div>
+                    <div className={styles.long_text_box}>
+                        <label>email</label>
+                        <input type="text" className="form-control"></input>
+                    </div>
+                    <div className={styles.col_cont}>
+                        <div className={styles.col_items}>
+                            <label>gender</label>
+                            <select className="form-control">
+                                <option value="" disabled selected hidden>select gender&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;⌄</option>
```
Comment (thomaspwang) [inline src/pages/Signup/Signup.js]: LOL, this is really jank. Instead, add an SVG logo and align it to the right of the input box.

**Review by thomaspwang — CHANGES_REQUESTED** [review]
Amazing! Looks great. Just change one thing about the dropdown logo :)

**Review by DylanHopkins21 — APPROVED** [review]
Looks good to me! Made tiny nitpicky change in the CSS and didn't feel the need to make you go change it

**Review by thomaspwang — APPROVED** [review]
Looks good to me! (after you fix SVGs)

**Comment by brunobmv** [comment]
Screen Shot 2023-03-25 at 20 10 51

**Pull request merged**